spec/index.bs

```diff
@@ -192,34 +192,82 @@ dictionary GPULimits {
 </script>
 
 
-Buffers {#buffers}
-==================
+# {{GPUBuffer}} # {#GPUBuffer}
 
-## GPUBuffer ## {#buffer}
+A {{GPUBuffer}} represents a block of memory that can be used in GPU operations.
+Data is stored in linear layout, meaning that each byte of the allocation can be
+addressed by its offset from the start of the {{GPUBuffer}}, subject to alignment
+restrictions depending on the operation..
 
-<script type=idl>
-interface GPUBuffer : GPUObjectBase {
-    Promise<ArrayBuffer> mapReadAsync();
-    Promise<ArrayBuffer> mapWriteAsync();
-    void unmap();
+{{GPUBuffer}} has the following internal slots:
 
-    void destroy();
-};
-</script>
+<dl dfn-type=attribute dfn-for="GPUBuffer">
+	: <dfn>\[[size]]</dfn>
+	::
+        The Length of the {{GPUBuffer}} allocation in bytes, which is an unsigned long long.
+
+	: <dfn>\[[usage]]</dfn>
```

Comment:
nit: instead of "represented as a bitset unsigned long", can just say `of type {{GPUBufferUsageFlags}}, readonly`.